docusaurus/docs/operate/configs/supplier_staking_config.md

```diff
@@ -54,10 +94,60 @@ _`Required`_, _`Non-empty`_
 owner_address: <address>
 ```
 
-The `owner_address` is used as the unique shareholder address for the `Supplier`
+The `owner_address` is the address of the account that owns the `Supplier`s staked
+funds, which will be returned to this account when the `Supplier` unstakes and finishes
+unbonding.
+
+For custodial staking, the `owner_address` is the same as the `operator_address`.
+
+For non-custodial staking, the `owner_address` must be different from the `operator_address`.
+This address receives back the staked `upokt` when the `Supplier` is unstaked.
+
+The `owner_address` can only be changed with a stake message signed by the
+`Supplier`'s owner account.
+
+It is also used as the unique shareholder address for the `Supplier`
 if none of `default_rev_share_percent` or `rev_share_percent` is defined in the
 configuration file.
 
+:::note
+
+The `owner_address` does not identify a `Supplier`; multiple `Supplier`s can have
+the same `owner_address`.
+
+:::
+
+### `operator_address`
+
+_`Optional`_, _`Non-empty`_
+
+```yaml
+operator_address: <address>
+```
+
+The `operator_address` is the address that identifies the `Supplier`.
+Its corresponding account is used for operational tasks such as signing `RelayResponse`s,
+submitting `Claims` and `Proofs` as well as updating the `Supplier`'s info
+(i.e. Adding or removing services, increasing the stake amount, etc).
+
+The operator account can also be used to unstake the `Supplier` and send the staked
+`upokt` back to the `owner_address`.
+
+If the `operator_address` is not specified, the `owner_address` is used as the
```

Comment:
Does "not specified" mean that the field is not present, that it is an empty string, or either?

Reply:
An omitted `operator_address` results in an empty string when parsed. So it's either.

Will update it.